pandas/tests/groupby/test_groupby.py

```diff
@@ -2419,3 +2419,27 @@ def test_rolling_wrong_param_min_period():
     result_error_msg = r"__init__\(\) got an unexpected keyword argument 'min_period'"
     with pytest.raises(TypeError, match=result_error_msg):
         test_df.groupby("name")["val"].rolling(window=2, min_period=1).sum()
+
+
+@pytest.mark.parametrize("method", ["first", "last", "nth"])
+def test_groupby_last_first_nth_with_none(method):
```

Comment:
Please define expected without applying a method and then parametrize over None and nan

Reply:
I don't quite understand what you mean: the issue states that `None` and `np.nan` should lead to the same result when I `groupby` and apply a method. If I don't apply a method, then what's the point of the test? My reasoning was: I have two `Series` with `None` and `np.nan` and I have to test that grouping with a method gives the same output.

Reply:
Ok, I think I get what you mean